spec/resources/verification_request_spec.rb

```diff
@@ -109,6 +109,45 @@
     end
   end
 
+  describe '.cancel' do
+    let(:id) { 'AAAAAAAAQnIAAYd5YHFVOm8PNX2ecFbEjqV__upOKUE8YE_IK2Gw2CAN' }
+    subject { Truework::VerificationRequest.cancel(id, 'other', 'blah') }
+
+    context 'when successful' do
+      before do
+        stub_request(:put, "#{Truework.api_base}#{Truework::VerificationRequest.resource_path}#{id}/cancel/")
+          .to_return(
```

Comment:
can you add the mock to make sure the request is sending the correct body? I've also added in a couple other let's that we can also replace for these constants: https://github.com/truework/truework-sdk-ruby/pull/5/files#diff-c6f79b48d1bcb30b575eabbb970f7824R132

```
  describe '.cancel' do
    let(:id) { 'AAAAAAAAQnIAAYd5YHFVOm8PNX2ecFbEjqV__upOKUE8YE_IK2Gw2CAN' }
    let(:cancellation_reason) { 'other' }
    let(:cancellation_details) { 'The verification request is no longer needed.' }
    subject { Truework::VerificationRequest.cancel(id, cancellation_reason, cancellation_details) }

    context 'when successful' do
      before do
        stub_request(:put, "#{Truework.api_base}#{Truework::VerificationRequest.resource_path}#{id}/cancel/")
          .with(
            body: {
              'cancellation_reason' => cancellation_reason,
              'cancellation_details' => cancellation_details
            }
          )
          .to_return(
            status: 200,
            body: body
          )
      end
```